src/test_demo/updatedDetectionGreyScale.py

```diff
@@ -59,10 +65,19 @@
 
         frame = cv2.polylines(frame, [np.int32(dst)], True, (0, 255, 0), 3, cv2.LINE_AA)
         print("Match found!")
+        break
+    else:
+        print("Not matching")
 
     # Display the video feed with detected template
     cv2.imshow('Detected Template', frame_matches)
 
+
+    # 카메라가 10초가 지나면 종료
```

Comment:
As I mentioned before, you should change comment to English.

Reply:
I also changed it like before